.github/workflows/ci_cd.yml

```diff
@@ -137,6 +137,21 @@ jobs:
           name: docs
           path: api/build/html/
 
+      - name: Write logs
+        if: always()
```

Comment:
Is there a difference between `if: always()` and not having any conditional on these steps? I think there's something obvious here I'm missing, hah.

Reply:
Steps with `if` `always()` runs even if a previous step has failed, sort of like a `finally` in a `try`-`catch` block.

Reply:
Oh, that's really helpful—thanks!